frontend/src/components/pages/profile/Profile.tsx

```diff
@@ -55,17 +50,18 @@ export default function Profile() {
    */
   const contextServices = React.useContext(ServiceContext);
   const profileHandlerService = contextServices.profileHandlerService;
+  const loginStatusHandlerService = contextServices.loginStatusHandlerService;
 
-  const [person, setPerson] = React.useState<PersonProps|undefined>(undefined);
-  const [profileId, setProfileId] = React.useState<string>(undefined);
+  const [person, setPerson] = React.useState<PersonInterface | undefined>(undefined);
   const [submitSuccess, setSubmitSuccess] = React.useState<boolean|undefined>(undefined);
 
   React.useEffect(() => {
     (async() => {
-      const personPromise = await profileHandlerService.getPerson(USER_ID);
-      setPerson(personPromise);
+      const personResponse = await profileHandlerService.getPerson(
```

Comment:
This could potentially run pretty frequently (by default useEffect executes whenever any part of the component is re-rendered). Is there a way we could add better caching here?

Reply:
This useEffect should only one once upon mounting of component because we have an empty array as the second argument. 

Relevant passage: "If you want to run an effect and clean it up only once (on mount and unmount), you can pass an empty array ([]) as a second argument. This tells React that your effect doesn’t depend on any values from props or state, so it never needs to re-run. This isn’t handled as a special case — it follows directly from how the dependencies array always works." 
Article here: [React Documentation](https://reactjs.org/docs/hooks-effect.html)